### PR TITLE
PE-1972: protect all arfs tags

### PR DESCRIPTION
**Pull request by fedellen**

### CODE REVIEW

**Review by matibat — APPROVED** [review]
Perfect!

File: src/types/custom_metadata_type.test.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { expect } from 'chai';
+import { gqlTagNameArray } from '../utils/constants';
+import { isCustomMetaDataGqlTags } from './custom_metadata_types';
+
+describe('isCustomMetaDataGqlTags type guard function', () => {
+	describe('returns false if provided tag name collides with any protected ArFS GQL tag name', () => {
+		for (const tagName of gqlTagNameArray) {
```
Comment (arielmelendez) [inline src/types/custom_metadata_type.test.ts:7]: This is an example of the test code using app code in a way that could lead to false positives in our tests. What do you think about keeping a separate list here and have a test that verifies that the app's constant has the same values and array length? Then tests will break when someone adds or removes something from the app code's list.

Comment (fedellen) [reply src/types/custom_metadata_type.test.ts:7]: Ah, thats completely right

Comment (fedellen) [reply src/types/custom_metadata_type.test.ts]: 1c03446

**Review by matibat — APPROVED** [review]
LGTM

**Review by arielmelendez — APPROVED** [review]

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**